FEM/OutputTools.h

```diff
@@ -65,16 +65,16 @@ struct ELEMENT_MMP_VALUES
 
 struct ELEMENT_MFP_VALUES
 {
-	static double getValue(CFluidProperties* mfp, int mfp_id)
+	static double getValue(CFluidProperties* mfp, int mfp_id, double* var_values = NULL)
 	{
 		double mat_value = .0;
 		switch (mfp_id)
 		{
 			case 0:
-				mat_value = mfp->Density();
+				mat_value = mfp->Density(var_values);
```

Comment:
If I remember it correctly, when the state variables are not passed, `Density()` will try to get them within the function. In that case, why do you need to pass it from outside?